app/services/encrypted_email.rb

```diff
@@ -30,7 +30,7 @@ def decrypted
   end
 
   def fingerprint
-    Pii::Fingerprinter.fingerprint(email)
+    Pii::Fingerprinter.fingerprint(email.downcase)
```

Comment:
are we consistent about downcasing emails? also, should we be? domains are case-insensitive, but email providers are allowed to have case-insensitive usernames IIRC